packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -142,11 +142,11 @@ BuildApp() {
     mkdir "${derived_dir}/engine"
     RunCommand cp -r -- "${flutter_podspec}" "${derived_dir}/engine"
     RunCommand cp -r -- "${flutter_framework}" "${derived_dir}/engine"
-    RunCommand find "${derived_dir}/engine/Flutter.framework" -type f -exec chmod a-w "{}" \;
```

Comment:
Maybe add a comment noting this is marking all non-binary files as read-only so that we can code-sign the binaries.

Reply:
I also wonder if there's any way to get the same list using only options to `find` rather than feeding it through `grep -Il` then into `xargs`. 

Looking at the manpage, I don't see anything promising though.

Reply:
That was my first thought but I couldn't find any such flag.